.eslintrc

```diff
@@ -0,0 +1,62 @@
+{
+  "ecmaFeatures": {
+    "jsx": true
+  },
+  "env": {
+    "browser": true,
+    "node": true
+  },
+  "rules": {
+    // Warn when double quotes are used.
+    "quotes": [1, "single"],
+
+    // Warn when trailing spaces are detected.
+    "no-trailing-spaces": 1,
+
+    // Warn when dangling commas are present.
+    "no-comma-dangle": 1,
+
+    // Warn when the brace style isn't correct.
+    "brace-style": [1, "1tbs"],
+
+    // Disable reporting on missing "use strict" statements. By virtue of
+    // having at least one dependency that has "use strict" enabled, we
+    // get it for free.
+    "strict": [0, "function"],
+
+    // Turn off error/warning for dangling underscores. We're ok with them.
+    "no-underscore-dangle": 0,
+
+    // Warn if a global variable is being overwritten.
+    "no-undef": 1,
+
+    // Warn if a variable in a function/method shadows another variable defined
+    // at a higher scope.
+    "no-shadow": 1,
+
+    // Warn if there are unused variables in a file.
+    "no-unused-vars": 1,
+
+    // Warn if there are extraneous spaces on a line.
+    "no-multi-spaces": 1,
+
+    // Warn if the are extraneous space characters around a key-value pair
+    // in an object literal.
+    "key-spacing" : 1,
+
+    // Error if the something other than two spaces is used for indentation.
+    "indent": [1, 2],
+
+    // Warn if the cyclomatic complexity goes above six for any given function.
+    "complexity": [1, 6],
+
```

Comment:
Maybe I'm just being an a-hole at this point but:
Recommend `"space-in-parens": 1`